contacts/html-helper.js

```diff
@@ -181,11 +189,22 @@ const populateFavouritesListHTML = () => {
   });
 };
 
+export const changeClearButtonState = () => {
+  setButtonStatus(
+    getElementById(DOCUMENT_ELEMENTS.FAVOURITES.CLEAR_ALL_BUTTON),
+    ContactsService.arePersonsWithLikes()
+  );
+};
+
 export const reRenderFavouritesListHTML = () => {
+  const arePersonsWithLikes = ContactsService.arePersonsWithLikes();
+  // repopulate
   clearFavouritesListHTML();
-  if (ContactsService.arePersonsWithLikes()) {
+  if (arePersonsWithLikes) {
```

Comment:
and particular reason why you created arePersonsWithLikes? if no specific reason, you can leave it as it was initially

Reply:
initially I've used it twice in the function, but when I modified the function body I forgot it there